toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/internal/ScalebarRenderer.kt

```diff
@@ -415,6 +513,30 @@ internal fun BarScaleBarPreview() {
     }
 }
 
+@Preview(showBackground = true, backgroundColor = 0xff91d2ff)
+@Composable
+internal fun DualUnitLineScalebarPreview() {
+    val maxWidth = 175.dp
+    val displayLength = 139.3
+    val endScalebarDivision = ScalebarDivision(displayLength,"3000 m")
+    val alternateScalebarDivision = ScalebarDivision(0.75 * displayLength,"3500 Km")
```

Comment:
these values should be flipped sine the primary label is always bigger than the secondary label
```suggestion
    val endScalebarDivision = ScalebarDivision(displayLength,"3500 mi")
    val alternateScalebarDivision = ScalebarDivision(0.75 * displayLength,"3000 Km")
```

Reply:
No, not necessarily
![Screenshot_20250203_125328](https://github.com/user-attachments/assets/347f2687-2038-45ef-b60e-59e1bc50c514)